README.md

```diff
@@ -18,6 +18,14 @@ It generates the cache files using the [QueuedJobs module](https://github.com/sy
 
 [Docs](docs/en/index.md)
 
+## Requirements
+
+* "silverstripe/framework": "^5",
+* "silverstripe/cms": "^5",
+* "silverstripe/config": "^2",
+* "symbiote/silverstripe-queuedjobs": "^5",
+* "silverstripe/versioned": "^2"
```

Comment:
```suggestion
```
That's all in the composer.json, doesn't need to be repeated here where it's likely to get outdated.